README.md

```diff
@@ -435,6 +434,49 @@ pool.exec('eventExample', [], {
 });
 ```
 
+### Signals
+You can optionally allow a `AbortController` instance to be created when executing in a worker. Each `task` is given a unique controller instance which is `aborted` if there is an error condition within the worker execution. Allowing the `onabort` event to be listened to for operations which might need clean up.
+
+Example:
+
+```js
+// file: myWorker.js
+var workerpool = require('../../');
+
+function abort(signal) {
+    return new Promise(function (_resolve, reject) {
+      signal.addEventListener("abort", () => {
+          workerpool.workerEmit({
```

Comment:
Can you change this part `workerpool.workerEmit({` with a `console.log()`, so it is clear that nothing is "happening" here?